quadhoral-openmeteo/quadhoral_lambda.py

```diff
@@ -0,0 +1,30 @@
+'''Orchestrates weekly ETL pipeline'''
+# pylint: disable=W0612,W0613,E0606
+import logging
+from dotenv import load_dotenv
+from quadhoral_extract import extract
+from quadhoral_transform import transform
+from quadhoral_load import load
+
+def lambda_handler(event, context):
+    '''Orchestrates ETL pipeline for weekly OpenMeteo data'''
```

Comment:
Nice logging implementation so it's clear what is happening across the key stages of the quadhoural weather pipeline. There will be AWS logs for Lambdas with CloudWatch, but this gives a more granular logging in the terminal about what specific part of the pipeline potentially failed.